parent/pom.xml

```diff
@@ -154,7 +154,7 @@
     <version.springdoc>2.7.0</version.springdoc>
     <version.jakarta.json>2.0.1</version.jakarta.json>
 
-    <version.node>v20.12.2</version.node>
+    <version.node>v22.21.0</version.node>
```

Comment:
let's leep v20 (just higher minor), otherwise we need another review from FE engineer.